tools/slang-unit-test/unit-test-rtti.cpp

```diff
@@ -0,0 +1,74 @@
+// unit-test-rtti.cpp
+
+#include "../../source/core/slang-rtti-info.h"
+
+#include "tools/unit-test/slang-unit-test.h"
+
+using namespace Slang;
+
+namespace { // anonymous 
+
+struct SomeStruct
+{
+    int a = 0;
+    float b = 2.0f;
+    String s;
+    List<String> list;
+
+    static const StructRttiInfo g_rttiInfo;
+};
+
+} // anonymous
+
+static const StructRttiInfo _makeSomeStructRtti()
```

Comment:
I guess it is intended to have some build tool that generates these functions in the future?

Reply:
Yes it would be better if we could generate these automatically, as they are rather tedious to write. The C++ extractor could do this, but would need some way to convey some of the options appropriately.